pubspec.yaml

```diff
@@ -6,8 +6,11 @@ environment:
   sdk: '>=1.20.1 <2.0.0'
 
 dependencies:
+  analysis_server_lib: ^0.1.3
```

Comment:
If this is where the code to interface with the server lives, then we should unify it with 'analysis_server_client'.

Reply:
Sounds good. This is at least a library that I'm familiar with, which is why I used it here.